aten/src/ATen/native/cpu/BinaryOpsKernel.cpp

```diff
@@ -116,8 +116,8 @@ void div_kernel(TensorIterator& iter) {
 }
 
 void remainder_kernel(TensorIterator& iter) {
-  if (isIntegralType(iter.dtype(), /*includeBool*/ false)) {
-    AT_DISPATCH_INTEGRAL_TYPES(iter.dtype(), "remainder_cpu", [&]() {
+  if (isIntegralType(iter.common_dtype(), /*includeBool*/ true)) {
```

Comment:
Like with fmod, let's exclude bool for remainder, too. NumPy also promotes bool inputs to remainder.